demos/controlgroup/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Nope. These were added by @tjvantoll in 343c2651729af2fdf020dd538fce0855785f5a0d.